### PR TITLE
change query filters so only courses from current quarter shows up

### PR DESCRIPTION
**Pull request by ashwin9798**
The change was tested by creating a course in Winter 2018 and a course in Summer 2018, and then attempting to enroll. In the dropdown, only the Winter 2018 course appeared.

### CODE REVIEW

File: embed_grader/serapis/forms/course_forms.py [hunk 1]
```diff
@@ -119,14 +119,19 @@ class CourseEnrollmentForm(Form):
...
     def __init__(self, *args, **kwargs):
         self.user = kwargs.pop('user')
         super(CourseEnrollmentForm, self).__init__(*args, **kwargs)
 
-        current_year = timezone.now().year
+        now = timezone.now()
+        cur_year = now.year
+        cur_month = now.month
+        cur_quarter = (cur_month - 1) // 3
+        cur_year_quarter = cur_year * 4 + cur_quarter
```
Comment (TimeString) [inline embed_grader/serapis/forms/course_forms.py]: This line seems redundant

Comment (ashwin9798) [reply embed_grader/serapis/forms/course_forms.py]: which line is redundant exactly?

Comment (TimeString) [reply embed_grader/serapis/forms/course_forms.py]: 131

Comment (ashwin9798) [reply embed_grader/serapis/forms/course_forms.py]: oh right, it can be removed.

Comment (TimeString) [reply embed_grader/serapis/forms/course_forms.py]: 👍

Comment (ashwin9798) [reply embed_grader/serapis/forms/course_forms.py]: I have just added a description of the test.

Comment (ashwin9798) [reply embed_grader/serapis/forms/course_forms.py]: The change was tested by creating a course in Winter 2018 and a course in Summer 2018, and then attempting to enroll. In the dropdown, only the Winter 2018 course appeared.

**Pull request merged**